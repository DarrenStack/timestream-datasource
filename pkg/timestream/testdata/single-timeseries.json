{
    "ColumnInfo": [
        {
            "Name": "region",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "cell",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "silo",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "availability_zone",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "microservice_name",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "instance_name",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "process_name",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "jdk_version",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": "VARCHAR",
                "TimeSeriesMeasureValueColumnInfo": null
            }
        },
        {
            "Name": "gc_reclaimed",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": null,
                "TimeSeriesMeasureValueColumnInfo": {
                    "Name": null,
                    "Type": {
                        "ArrayColumnInfo": null,
                        "RowColumnInfo": null,
                        "ScalarType": "DOUBLE",
                        "TimeSeriesMeasureValueColumnInfo": null
                    }
                }
            }
        }
    ],
    "NextToken": null,
    "QueryId": "#QueryId#",
    "Rows": [
        {
            "Data": [
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1-cell-5",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1-cell-5-silo-2",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "ap-northeast-1-3",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "zeus",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "i-zaZswmJk-zeus-0002.amazonaws.com",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "server",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": "JDK_11",
                    "TimeSeriesValue": null
                },
                {
                    "ArrayValue": null,
                    "NullValue": null,
                    "RowValue": null,
                    "ScalarValue": null,
                    "TimeSeriesValue": [
                        {
                            "Time": "2020-08-20 21:31:18.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "99.45151776944134",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:34:41.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "63.351232308040025",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:38:04.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "80.87701379376135",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:41:25.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "11.433718482140609",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:44:48.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "50.91424854459535",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:48:15.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "74.06796865135388",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:51:41.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "48.800300503120106",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:55:07.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "95.88258548308211",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 21:58:33.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "9.585793416959621",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:02:00.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "51.28941845830609",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:05:26.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "47.04617229244753",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:08:51.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "21.81747242060963",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:12:17.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "58.27729852682339",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:15:47.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "9.72475521954138",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:19:16.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "61.83377906091083",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:22:41.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "37.58059705439627",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:25:52.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "11.997284953211562",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:29:04.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "42.22641777489885",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:32:17.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "2.722492507358776",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:35:30.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "86.24076392157414",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:38:44.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "81.42162184752405",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:42:03.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "98.11275131328703",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:46:28.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "40.99461156340832",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:49:22.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "91.72618576627222",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:52:15.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "52.41397322170659",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:55:07.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "55.478198466465635",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 22:58:00.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "21.91462915061515",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:00:53.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "87.30016945532378",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:03:45.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "86.65370722452221",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:06:39.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "97.93780275397435",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:09:32.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "31.027125724662042",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:12:26.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "38.87760832447908",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:15:21.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "75.24858076058761",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:18:15.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "66.96363134954842",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:21:09.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "46.916193735129056",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:24:03.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "0.7487980431854391",
                                "TimeSeriesValue": null
                            }
                        },
                        {
                            "Time": "2020-08-20 23:26:57.000000000",
                            "Value": {
                                "ArrayValue": null,
                                "NullValue": null,
                                "RowValue": null,
                                "ScalarValue": "83.05828218549377",
                                "TimeSeriesValue": null
                            }
                        }
                    ]
                }
            ]
        }
    ]
}