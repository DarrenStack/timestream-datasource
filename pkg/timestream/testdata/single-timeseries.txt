Frame[0] {
    "custom": {
        "hasSeries": true,
        "queryId": "{CHANGES}"
    },
    "stats": []
}
Name: 
Dimensions: 2 Fields by 55 Rows
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Name: time                    | Name: gc_reclaimed                                                                                                                                                                                                                                  |
| Labels:                       | Labels: availability_zone=ap-northeast-1-3, cell=ap-northeast-1-cell-5, instance_name=i-zaZswmJk-zeus-0002.amazonaws.com, jdk_version=JDK_11, microservice_name=zeus, process_name=server, region=ap-northeast-1, silo=ap-northeast-1-cell-5-silo-2 |
| Type: []time.Time             | Type: []*float64                                                                                                                                                                                                                                    |
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| 2020-06-06 01:34:15 +0000 UTC | 62.23581138845643                                                                                                                                                                                                                                   |
| 2020-06-06 01:36:29 +0000 UTC | 59.49083570430662                                                                                                                                                                                                                                   |
| 2020-06-06 01:38:43 +0000 UTC | 44.05578858462976                                                                                                                                                                                                                                   |
| 2020-06-06 01:40:58 +0000 UTC | 97.80615601471405                                                                                                                                                                                                                                   |
| 2020-06-06 01:43:15 +0000 UTC | 62.20460105243668                                                                                                                                                                                                                                   |
| 2020-06-06 01:45:30 +0000 UTC | 35.48482795335563                                                                                                                                                                                                                                   |
| 2020-06-06 01:47:45 +0000 UTC | 82.7521637310198                                                                                                                                                                                                                                    |
| 2020-06-06 01:49:57 +0000 UTC | 16.372119218328464                                                                                                                                                                                                                                  |
| 2020-06-06 01:52:06 +0000 UTC | 67.86206920047387                                                                                                                                                                                                                                   |
| ...                           | ...                                                                                                                                                                                                                                                 |
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+



