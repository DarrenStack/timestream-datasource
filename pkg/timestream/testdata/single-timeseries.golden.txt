ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {}
Name: 
Dimensions: 2 Fields by 37 Rows
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Name: time                    | Name: gc_reclaimed                                                                                                                                                                                                                                  |
| Labels:                       | Labels: availability_zone=ap-northeast-1-3, cell=ap-northeast-1-cell-5, instance_name=i-zaZswmJk-zeus-0002.amazonaws.com, jdk_version=JDK_11, microservice_name=zeus, process_name=server, region=ap-northeast-1, silo=ap-northeast-1-cell-5-silo-2 |
| Type: []time.Time             | Type: []*float64                                                                                                                                                                                                                                    |
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| 2020-08-20 21:31:18 +0000 UTC | 99.45151776944134                                                                                                                                                                                                                                   |
| 2020-08-20 21:34:41 +0000 UTC | 63.351232308040025                                                                                                                                                                                                                                  |
| 2020-08-20 21:38:04 +0000 UTC | 80.87701379376135                                                                                                                                                                                                                                   |
| 2020-08-20 21:41:25 +0000 UTC | 11.433718482140609                                                                                                                                                                                                                                  |
| 2020-08-20 21:44:48 +0000 UTC | 50.91424854459535                                                                                                                                                                                                                                   |
| 2020-08-20 21:48:15 +0000 UTC | 74.06796865135388                                                                                                                                                                                                                                   |
| 2020-08-20 21:51:41 +0000 UTC | 48.800300503120106                                                                                                                                                                                                                                  |
| 2020-08-20 21:55:07 +0000 UTC | 95.88258548308211                                                                                                                                                                                                                                   |
| 2020-08-20 21:58:33 +0000 UTC | 9.585793416959621                                                                                                                                                                                                                                   |
| ...                           | ...                                                                                                                                                                                                                                                 |
+-------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////2AIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAvP3//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADc/f//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAPz9//8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAgAAAMABAAAYAAAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAGwBAABsAQAAAAADAWwBAAACAAAANAAAAAQAAABg/v//CAAAABgAAAAMAAAAZ2NfcmVjbGFpbWVkAAAAAAQAAABuYW1lAAAAAIz+//8IAAAAEAEAAAYBAAB7ImF2YWlsYWJpbGl0eV96b25lIjoiYXAtbm9ydGhlYXN0LTEtMyIsImNlbGwiOiJhcC1ub3J0aGVhc3QtMS1jZWxsLTUiLCJpbnN0YW5jZV9uYW1lIjoiaS16YVpzd21Kay16ZXVzLTAwMDIuYW1hem9uYXdzLmNvbSIsImpka192ZXJzaW9uIjoiSkRLXzExIiwibWljcm9zZXJ2aWNlX25hbWUiOiJ6ZXVzIiwicHJvY2Vzc19uYW1lIjoic2VydmVyIiwicmVnaW9uIjoiYXAtbm9ydGhlYXN0LTEiLCJzaWxvIjoiYXAtbm9ydGhlYXN0LTEtY2VsbC01LXNpbG8tMiJ9AAAGAAAAbGFiZWxzAAAAAAAAgv///wAAAgAMAAAAZ2NfcmVjbGFpbWVkAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAAACkwAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABAAAAHRpbWUAAAAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAEAAAAdGltZQAAAAAAAAAA/////7gAAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAABQAgAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAABYAAAAJQAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACgBAAAAAAAAKAEAAAAAAAAAAAAAAAAAACgBAAAAAAAAKAEAAAAAAAAAAAAAAgAAACUAAAAAAAAAAAAAAAAAAAAlAAAAAAAAAAAAAAAAAAAAAPzMqVAXLRYAKovtfxctFgBYSTGvFy0WAPLR/d0XLRYAIJBBDRgtFgB2uXM9GC0WAAJIam0YLRYAjtZgnRgtFgAaZVfNGC0WAHCOif0YLRYA/ByALRktFgC+EDtdGS0WAEqfMY0ZLRYA/pgWvhktFgDo97/uGS0WAKrreh4aLRYAYGjzShotFgDgf6d3Gi0WACoyl6QaLRYAdOSG0RotFgCIMbL+Gi0WAI6EBy0bLRYAqL+6ahstFgD09D2TGy0WAHaPhbsbLRYALo+R4xstFgCwKdkLHC0WADLEIDQcLRYA6sMsXBwtFgA2+a+EHC0WALiT96wcLRYABMl61RwtFgAamTn+HC0WAGbOvCYdLRYAsgNATx0tFgD+OMN3HS0WAEpuRqAdLRYPVMmq5dxYQE4qJi71rE9AI5Z2/iA4VEDq7VBZEN4mQPC5pxgGdUlA9q0vmVmEUkDF7zM/cGZIQMVw0kd8+FdAehtgHe0rI0BGpP6pC6VJQJEFQ/nohUdAe+Zf30XRNUC896OEfiNNQDESux0TcyNAzlmzRbnqTkCZZBgBUcpCQFCsJSKc/idAOjb1QfscRUAXsNUmqscFQO5NFK1oj1VARpkz2vtaVECHyUhRN4dYQPWHhG5Pf0RAbz3d03nuVkAlUxQT/TRKQM87e5s1vUtAPtzRIiXqNUDsd/L5NdNVQNae01bWqVVAyprX9QR8WEDETSS28QY/QOEqNnhVcENA30hHv+jPUkB+7dIirL1QQPaWGNZFdUdAyElZUCf25z+mKDTlusNUQBAAAAAMABQAEgAMAAgABAAMAAAAEAAAACwAAAA4AAAAAAADAAEAAADoAgAAAAAAAMAAAAAAAAAAUAIAAAAAAAAAAAAAAAAAAAAACgAMAAAACAAEAAoAAAAIAAAAdAAAAAMAAABMAAAAKAAAAAQAAAC8/f//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAANz9//8IAAAADAAAAAAAAAAAAAAABAAAAG5hbWUAAAAA/P3//wgAAAAMAAAAAgAAAHt9AAAEAAAAbWV0YQAAAAACAAAAwAEAABgAAAAAABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAbAEAAGwBAAAAAAMBbAEAAAIAAAA0AAAABAAAAGD+//8IAAAAGAAAAAwAAABnY19yZWNsYWltZWQAAAAABAAAAG5hbWUAAAAAjP7//wgAAAAQAQAABgEAAHsiYXZhaWxhYmlsaXR5X3pvbmUiOiJhcC1ub3J0aGVhc3QtMS0zIiwiY2VsbCI6ImFwLW5vcnRoZWFzdC0xLWNlbGwtNSIsImluc3RhbmNlX25hbWUiOiJpLXphWnN3bUprLXpldXMtMDAwMi5hbWF6b25hd3MuY29tIiwiamRrX3ZlcnNpb24iOiJKREtfMTEiLCJtaWNyb3NlcnZpY2VfbmFtZSI6InpldXMiLCJwcm9jZXNzX25hbWUiOiJzZXJ2ZXIiLCJyZWdpb24iOiJhcC1ub3J0aGVhc3QtMSIsInNpbG8iOiJhcC1ub3J0aGVhc3QtMS1jZWxsLTUtc2lsby0yIn0AAAYAAABsYWJlbHMAAAAAAACC////AAACAAwAAABnY19yZWNsYWltZWQAABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAARAAAAEwAAAAAAAAKTAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAABAAAAAEAAAAdGltZQAAAAAEAAAAbmFtZQAAAAAAAAAAAAAGAAgABgAGAAAAAAADAAQAAAB0aW1lAAAAAAADAABBUlJPVzE=
